super_scoring/src/main/java/ru/hh/super_scoring/config/ProdConfig.java

```diff
@@ -0,0 +1,18 @@
+package ru.hh.super_scoring.config;
```

Comment:
```suggestion
package ru.hh.superscoring.config;
```
переименовать по конвенции